tb_pulumi/cloudfront.py

```diff
@@ -154,6 +155,7 @@ def __init__(
                 'cached_methods': ['HEAD', 'GET'],
                 'cache_policy_id': CACHE_POLICY_ID_OPTIMIZED,
                 'compress': True,
+                'function_associations': default_function_associations,
```

Comment:
These changes allow us to associate CloudFront functions with Distributions' default behaviors.